commands/files/is_hidden.go

```diff
@@ -0,0 +1,19 @@
+// +build !windows
+
+package files
+
+import (
+	"path/filepath"
+	"strings"
+)
+
+func IsHidden(f File) bool {
+
+	fName := filepath.Base(f.FileName())
+
+	if strings.HasPrefix(fName, ".") && len(fName) > 1 {
```

Comment:
So I'm assuming that `filepath.Base(f.FileName())` will wipe out any instance of the "double dot" because we're just looking at the last element of the Path. http://golang.org/pkg/path/filepath/#Base

According to the docs, we only get a file name or a "single dot" for an empty path, hence the `len(fName) > 1` check.